crates/bevy_sprite/src/picking_backend.rs

```diff
@@ -114,7 +114,10 @@ fn sprite_picking(
         let Ok(cursor_ray_world) = camera.viewport_to_world(cam_transform, pos_in_viewport) else {
             continue;
         };
-        let cursor_ray_len = cam_ortho.far - cam_ortho.near;
+        let cursor_ray_len = cam_ortho
+            .far
+            .unwrap_or(OrthographicProjection::DEFAULT_FAR_PLANE)
```

Comment:
This is definitely wrong and I'm open to suggestions. Would also allow `DEFAULT_FAR_PLANE` not to be `pub`.